docker/nmslib/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.9"
+
+services:
+  okpt:
+    image: okpt/nmslib
```

Comment:
Should image name be parametrized and default to this?

Reply:
That makes sense, I've gone ahead and parameterized them with env variables with `okpt/...` as the default value.

Reply:
Awesome, this may be impacted by docker-compose build.